From 8170fffa7bad33c67b7c59ef5d98d92c0c742e91 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <perex@perex.cz>
Date: Wed, 9 Feb 2022 16:19:31 +0100
Subject: [PATCH] sof-hda-dsp: fix multiple If.dmic blocks
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix multiple If.dmic blocks - rename the first to devdmic
initialization.

Fixes: https://github.com/alsa-project/alsa-ucm-conf/pull/132
Reported-by: Kacper Michajłow <kasper93@gmail.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf b/ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf
index b47777b..eb243d1 100644
--- a/ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf
+++ b/ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf
@@ -3,7 +3,7 @@ Syntax 4
 Define.DeviceMic "Mic"
 Define.DeviceDmic ""
 
-If.dmic {
+If.devdmic {
 	Condition {
 		Type String
 		Haystack "${CardComponents}"
-- 
2.34.1

